colin/core/target.py

```diff
@@ -40,14 +57,19 @@ def _get_target_instance(target, logging_level):
         """
         Get the Container/Image instance for the given name.
         (Container is the first choice.)
+        or DockerfileParser instance if the target is path.
 
         :param target: str or instance of Image/Container
         :return: Container/Image
         """
         logger.debug("Finding target '{}'.".format(target))
 
         if isinstance(target, (Image, Container)):
+            logger.debug("Target is a conu object.")
             return target
+        if os.path.exists(target):
```

Comment:
Maybe more specifically `if os.path.isfile(target):`

Reply:
I don't know, why I always choose the wrong one..;-) Thanks